README.md

```diff
@@ -15,29 +15,26 @@ applications in an expressive way.
 
 ## Usage Documentation
 
-Usage documentation exists for each major version
+Usage documentation exists for each major version. Don't know what version you're on? You're probably using the version from the `master` branch, which is currently `v2`.
 
-- `v1` - [./docs/v1/manual.md](./docs/v1/manual.md)
 - `v2` - [./docs/v2/manual.md](./docs/v2/manual.md)
+- `v1` - [./docs/v1/manual.md](./docs/v1/manual.md)
 
-## Installation
-
-Make sure you have a working Go environment.  Go version 1.10+ is supported.  [See
-the install instructions for Go](http://golang.org/doc/install.html).
+### Using `v2` releases
 
-### GOPATH
+**Warning**: `v2` is in a beta state.
```

Comment:
I guess it's not really a beta, but rather early days, as beta implies that people should use v1 for something stable, which I don't think is what we want.

Reply:
I'll think on the wording here for a bit, and stand up a PR for changing this line 👍